.github/workflows/ci.yml

```
@@ -0,0 +1,15 @@
+  biome:
+    name: runner / Biome
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      pull-requests: write
+    steps:
+      - uses: actions/checkout@v5
+      - uses: mongolyy/reviewdog-action-biome@v1
+        with:
+          github_token: ${{ secrets.github_token }}
+          reporter: github-pr-review
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)